xmtp_mls/src/groups/sync.rs

```diff
@@ -324,7 +324,12 @@ impl MlsGroup {
                 )
                 .await;
 
-                if maybe_validated_commit.is_err() {
+                if let Err(err) = maybe_validated_commit {
+                    log::warn!(
```

Comment:
I think it would be better to do `log::error` here, given we expect this to never happen, and the consequence is that we lose a commit

Reply:
Done